src/frontend/qt_sdl/LAN_Socket.cpp

```diff
@@ -216,6 +219,12 @@ bool Init()
 
     Ctx = slirp_new(&cfg, &cb, nullptr);
 
+    if (Config::EnableWifiPacketCapturing)
+    {
+        LAN_Capture::Prepare();
```

Comment:
why can't those hooks go into Platform.cpp?

Reply:
Platform.cpp is the right place to place them, but weird stuff happens to the captured TX packets when I load the pcap file in wireshark:

![image](https://user-images.githubusercontent.com/8304241/126882018-0abb4d8f-0792-48c4-8ed7-df1c6a70354b.png)

this shouldn't be too hard for me to figure out